### PR TITLE
Add flexpool documentation for integration

### PR DESCRIPTION
**Pull request by DatDraggy**
## Proposed change

Add documentation for the new `flexpool `integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/47995
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2372
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/flexpool.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@DatDraggy'
+ha_domain: flexpool
+ha_config_flow: true
+---
+
+The Flexpool integration allows you to keep track of your miner details page by adding sensors for workers and pool statistics.
+
+To add the Flexpool sensors to your installation, specify an Ethereum address to watch in the `configuration.yaml` file. You can also optionally provide a token name to retrieve and ERC-20 token balance. If no token is provided then the balance retrieved will be in ETH. You can also optionally provide the token contract address in case the token name is not found.
```
Comment (frenck) [inline source/_integrations/flexpool.markdown]: - `and ERC-20` -> `an ERC-20`

> specify an Ethereum address to watch in the `configuration.yaml` file

I don't think this is correct? As this integration uses a configuration flow, there is no YAML configuration involved.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/flexpool.markdown (hunk 1 above)
Comment (DatDraggy) [reply source/_integrations/flexpool.markdown]: Oh, yea, staged the change but didn't commit. I used the etherscan integration to get a template to work on. Sorry. Fixed.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**